engine/src/main/db/update/scriptPort.sql

```diff
@@ -1,5 +1,14 @@
+-- EN-1748
 ALTER TABLE pages_metadata_draft ADD COLUMN groupcode character varying(30);
 ALTER TABLE pages_metadata_online ADD COLUMN groupcode character varying(30);
 UPDATE pages_metadata_online SET groupcode = (SELECT pages.groupcode FROM pages WHERE pages_metadata_online.code = pages.code);
 UPDATE pages_metadata_draft SET groupcode = (SELECT pages.groupcode FROM pages WHERE pages_metadata_draft.code = pages.code);
 ALTER TABLE pages DROP groupcode;
+
+-- EN-1858
+INSERT INTO sysconfig (version, item, descr, config) VALUES ('production', 'jacms_resourceMetadataMapping', 'Mapping between resource Metadata and resource attribute fields', '<mapping>
```

Comment:
Will this load automatically for CDP or do we have something else to do when we give them 4.3.4?